proposal/prev-url.d.ts.md

```diff
@@ -209,5 +253,67 @@ nonCanonicalScheme?: string | string[];
 ```
 
 ```ts
-} // module
+} // Importer
+```
+
+## Embedded Protocol
+
+### `Importer`
+
+#### `non_canonical_scheme`
+
+The set of URL schemes that are considered *non-canonical* for this importer.
+This must be empty unless `importer.importer_id` is set.
```

Comment:
Can it be empty when `importer.importer_id` is set?

Reply:
Yes, it just means that no URL schemes are considered non-canonical.